jenkins-scripts/dsl/brew_release.dsl

```diff
@@ -315,3 +315,77 @@ bottle_job_hash_updater.with
   }
 }
 
+// -------------------------------------------------------------------
+// BREW remove bottles for specified formulae and their dependencies
+def remove_dependent_bottles_job = job("generic-release-homebrew_remove_dependent_bottles")
+OSRFUNIXBase.create(remove_dependent_bottles_job)
+OSRFCredentials.allowOsrfbuildToRunTheBuild(remove_dependent_bottles_job)
+
+remove_dependent_bottles_job.with
+{
+  properties {
```

Comment:
we probably don't need to set a priority to run things on built-in at this point. they are usually fast.

Reply:
https://github.com/gazebo-tooling/release-tools/pull/1390/commits/b8a253a5edd400d672062387ebc2fa4be1629a74